[docs]: Update Deploying with Netlify Functions

jsjoeio opened the pull request:
While going through [this section](https://www.apollographql.com/docs/apollo-server/deployment/netlify/#set-up-apollo-server), I couldn't get the GraphQL Playground to show up until I added `playground: true` on the ApolloServer options object (thanks to this [thread on Spectrum](https://spectrum.chat/zeit/general/apollo-server-get-query-missing~ae8d5ca5-2faf-4c8d-8097-c3f6e25cad56)).

I thought I'd add this to the docs so others don't get stuck.

abernix commented on docs/source/deployment/netlify.md:
```diff
@@ -82,7 +82,8 @@ const resolvers = {
   typeDefs,
-  resolvers
+  resolvers,
+  playground: true
```
Fwiw, Playground is disabled automatically in Apollo Server when `process.env.NODE_ENV` is set to `production` unless `playground: true` is set explicitly as you've done here.  I assume `NODE_ENV=production` is set for a deployed Netlify Function, which explains why you couldn't get it to display without this change.

Forgive my unfamiliarity with the exact Netfliy workflow but is there not a Netlify Functions local development experience that has `NODE_ENV=development` (or `NODE_ENV` unset entirely?)

jsjoeio replied:
Oh, I didn't know that. When I was walking through this, I was specifically trying to get the Playground to work on my local machine (not in production). 

Here's the blurb from the docs:
<img width="816" alt="image" src="https://user-images.githubusercontent.com/3806031/60897282-a440b980-a21c-11e9-9871-bc186f5c6a87.png">

So I don't know if adding `playground: true` is the best solution. We could either:

1) Tell the user to add `NODE_ENV=development` in the `package.json` for the `npm run start:lambda` command
2) Tell the user to add `playground: true` if they can't see the Playground while developing locally.

abernix replied:
I think I like the first option and would support that change!  Though if that fixes it, I am certainly curious what is setting `NODE_ENV` to `production` by default!

jsjoeio commented:
@abernix I changed the wording to explain it how we agreed. Let me know what you think.

I ended up doing [the tutorial](https://www.apollographql.com/docs/apollo-server/deployment/netlify/) from the beginning. 

And you were right - GraphQL Playground didn't start up until I added `NODE_ENV=development`. 

### Without setting `NODE_ENV`

<img width="1280" alt="Screen Shot 2019-07-09 at 8 08 00 PM" src="https://user-images.githubusercontent.com/3806031/60937930-db4bb500-a286-11e9-81fb-a01b63e2ff7c.png">

### Setting `NODE_ENV=development`

<img width="1280" alt="Screen Shot 2019-07-09 at 8 09 28 PM" src="https://user-images.githubusercontent.com/3806031/60937931-db4bb500-a286-11e9-8b81-aa566b1d0336.png">
